src/WebApi/Application/UseCases/QuestionSet/DeleteQuestionSet/DeleteQuestionSetInput.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.UseCases.QuestionSet.DeleteQuestionSet
```

Comment:
Same as with the other PR, please convert it to file scoped namespace. 
We should probably look if its possible to make that a default 🤔